Error: Cannot find module 'browser-sync'

Saya dapat error ni bila run `gulp watch`

```
KL-PT-065:nasihati izwan$ gulp watch
module.js:472
    throw err;
    ^

Error: Cannot find module 'browser-sync'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/izwan/Documents/Baliomega/nasihati/gulpfile.js:1:81)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

--- a/package.json
+++ b/package.json
@@ -5,22 +5,23 @@
   "main": "index.js",
   "dependencies": {
     "body-parser": "^1.15.2",
-    "browser-sync": "^2.18.5",
     "cookie-parser": "^1.4.3",
     "express": "^4.14.0",
+    "moment": "^2.17.1",
+    "mongodb": "^2.2.16",
+    "pug": "^2.0.0-beta6",
+    "request": "^2.79.0"
+  },
+  "devDependencies": {
+    "browser-sync": "^2.18.5",
     "gulp": "^3.9.1",
     "gulp-concat": "^2.6.1",
     "gulp-cssnano": "^2.1.2",
     "gulp-postcss": "^6.2.0",
     "gulp-rename": "^1.2.2",
-    "moment": "^2.17.1",
-    "mongodb": "^2.2.16",
     "postcss-cssnext": "^2.9.0",
-    "postcss-nested": "^1.0.0",
-    "pug": "^2.0.0-beta6",
-    "request": "^2.79.0"
+    "postcss-nested": "^1.0.0"
   },
-  "devDependencies": {},
   "scripts": {
     "start": "node app.js",
     "dev": "nodemon app.js",

